ecommerce-product-page-main/styled.css

```diff
@@ -200,98 +197,65 @@ body{
     background-color: hsl(26,100%,55%);
 }
 
-
-
-img.space{
-    margin-top: 12px;
-    width: 130px;
-}
-
-ul.list{
-
+.list {
     display: flex;
-    list-style: none;
+    text-decoration: none;
+    gap: 10px;
+    align-items: center;
 }
 
 
-
-ul.list li+li:before{
-
-    padding: 10px;
-    content: " ";
-}       
-
-ul.list li a{
-
+.list a{
     color: hsl(219, 9%, 45%);
     text-decoration: none;
-
 }
 
 
-ul.list li a:hover{
-    padding-bottom: 38px;
+.list a:hover{
     border-bottom: 3px solid hsl(26,100%,55%);
 }
 
 
 .cart{
-    max-width: 25%;
-    max-height: 25%;
-    margin-top: 20px;
-    margin-left: 220px;
-
+    width: 50px;
+    height:50px;
+    margin-right: 20px;
 }
 
 
 .avatar{
-
-    max-width:60%;
-    max-height: 60%;
-    margin-left: 50px;
-    margin-top: 10px;
+    width: 50px;
+    height:50px;
 }
 
 
 .containblock{
-
     display: flex;
     margin-top: 70px;
-
-
+    margin-left: 80px;
 }
 
 .block1{
-
     flex: 0 0 200px;
     display: flex;
     flex-direction: column;
-    margin-left: 80px;
-
-    
-
 }
 
 .containthumbs{
-
     display: flex;
-    width: 318px;
+    gap: 10px;
 }
 
-.containthumbs >div img{
+.containthumbs > img{
     border-radius: 5px;
-    width: 77%;
-    border: 2px solid hsl(28,100%,55%);
-    border: hidden;
-
+    width: 65px;
 }
 
-.containthumbs >div img:hover{
+.containthumbs > img:hover{
 
      opacity: 0.4;

```

Comment:
DId your own orange border show fully on mouse hover over the thumbnails?
cos mine was blurry as in i couldn't find a way to make the border line unaffected by the opacity